<launch>
    <node name="wrist_node" pkg="wrist_node" type="wrist_node.py">
        <param name="w1_addr" value="1" type="int" />
        <param name="w2_addr" value="2" type="int" />
        <param name="dynamixel_port" value="/dev/ttyUSB0" type="string" />
        <param name="baud_rate" value="1000000" type="int" />

        <param name="rotate_Kp" value="1.0" type="float" />
        <param name="rotate_Ki" value="0" type="float" />
        <param name="rotate_Kd" value="0" type="float" />

        <param name="angle_Kp" value="1.0" type="float" />
        <param name="angle_Ki" value="0" type="float" />
        <param name="angle_Kd" value="0" type="float" />
    </node>
</launch>
